### PR TITLE
DM-25324: xml 6.0.0 deployment at NCSA

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: develop-env/light/Dockerfile [hunk 1]
```diff
@@ -73,8 +73,7 @@ RUN yum -y --enablerepo=extras install epel-release \
     tcpdump && \
     yum clean all
 
-COPY lsst-ts.repo /tmp/lsst-ts.repo
-RUN cat /tmp/lsst-ts.repo >> /etc/yum.conf
+RUN cat /tmp/lsst-ts-nexus.repo >> /etc/yum.conf
 
 RUN yum install -y OpenSpliceDDS-6.9.0-1
```
Comment (mareuter) [inline develop-env/light/Dockerfile:78]: Are the OpenSplice versions getting out of date here?

File: Jenkinsfile.salobj [hunk 2]
```diff
@@ -49,7 +49,7 @@ pipeline {
             }
             steps {
                 script {
-                    dockerImageMaster = docker.build(dockerImageNameMaster, "--no-cache --network salobj_b${BUILD_NUMBER} --build-arg ts_config_ocs=master --build-arg ts_config_attcs=master --build-arg config_mtcalsys_v=master --build-arg ts_config_eas=master --build-arg ts_config_latiss=master --build-arg ts_config_mttcs=master --build-arg ts_config_atcalsys=master --build-arg ts_sal=master --build-arg ts_salobj=master --build-arg ts_xml=master --build-arg ts_idl=master --build-arg base_image_tag=${base_image_tag} ./develop-env/salobj/")
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply Jenkinsfile.salobj:52]: removed

File: Jenkinsfile.salobj [hunk 3]
```diff
@@ -59,7 +59,7 @@ pipeline {
             }
             steps {
                 script {
-                    dockerImageDevelop = docker.build(dockerImageNameDevelop, "--no-cache --network salobj_b${BUILD_NUMBER} --build-arg ts_config_ocs=develop --build-arg ts_config_attcs=develop --build-arg config_mtcalsys_v=develop --build-arg ts_config_eas=develop --build-arg ts_config_latiss=develop --build-arg ts_config_mttcs=develop --build-arg ts_config_atcalsys=develop --build-arg ts_sal=develop --build-arg ts_salobj=develop --build-arg ts_xml=develop --build-arg ts_idl=develop --build-arg base_image_tag=${base_image_tag} ./develop-env/salobj/")
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply Jenkinsfile.salobj:62]: removed

File: deploy-env/conda/Dockerfile [hunk 4]
```diff
@@ -1,48 +1,47 @@
...
-ENV PYTHON_BUILD_LOCATION=/home/saluser/miniconda3
-#ENV TS_IDL_DIR=/home/saluser/repos/ts_idl/
-#ENV TS_XML_VERSION=${xml_branch}
 
 USER root
 
 RUN adduser -u 1004 -m -s /bin/bash saluser
 
-COPY lsst-ts.repo /tmp/lsst-ts.repo
+RUN cat /tmp/lsst-ts-nexus.repo >> /etc/yum.conf
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply deploy-env/conda/Dockerfile]: oops... I was trying to fix this and broke it... thanks for catching it... fixed

File: deploy-env/conda/setup_sal_env.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+. ~/miniconda3/bin/activate
+
+PYTHON_BUILD_VERSION=$(find /home/saluser/miniconda3/ -name Python.h \
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply deploy-env/conda/setup_sal_env.sh:5]: uhm... you are right... we don;t need this for the deployment environment...

[the remark this answers is not in the document]
Comment (tribeiro) [reply deploy-env/conda/setup_sal_env.sh]: removed.

File: develop-env/base/Dockerfile [hunk 6]
```diff
@@ -26,8 +26,7 @@ RUN if [ $UID -eq 1000 ] && [ $GID -eq 1000 ]; then  \
 RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash - && \
     yum install -y nodejs && node --version
 
-COPY lsst-ts-nexus.repo /tmp/lsst-ts.repo
-RUN cat /tmp/lsst-ts.repo >> /etc/yum.conf
+COPY lsst-ts-nexus.repo /etc/lsst-ts.repo
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply develop-env/base/Dockerfile:29]: Fixed the deploy-env... This is a better way of doing it.

File: develop-env/light/Dockerfile (hunk 1 above)
Comment (tribeiro) [reply develop-env/light/Dockerfile:78]: This is an old experimental image. I still plan on finish developing it (some day)... :(

**Review by mareuter — APPROVED** [review]
Looks good now.

**Pull request merged**